SwiftLeeds/Views/About/AboutView.swift

```diff
@@ -12,6 +12,10 @@ struct AboutView: View {
     // TODO: Set these to the correct pages
     let venueURL = URL(string: "https://swiftleeds.co.uk/#venue")
     let codeOfConduct = URL(string: "https://swiftleeds.co.uk/conduct")
+    
+    //MARK: Mock data for image url
+    let foregroundImageUrl = URL(string: "https://cdn.profoto.com/cdn/053149e/contentassets/d39349344d004f9b8963df1551f24bf4/profoto-albert-watson-steve-jobs-pinned-image-original.jpg")!
```

Comment:
It would be best to support local images for the `FancyHeaderView` and then use pass in the placeholders we currently have in the asset catalogue - what do you think?

Reply:
Amended FancyHeaderView to accept Asset name or urls